content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -63,6 +63,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - There's a new method, **List Threads** ([Web](/messaging/agent-chat-api/v3.2/#list-threads) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#list-threads)).
 - There's a new push, [**chat_unfollowed**](/messaging/agent-chat-api/v3.2/rtm-reference/#chat_unfollowed), complementary to the **Unfollow Chat** method.
 - The **Get Chat Threads Summary** method was removed.
+- The `restricted_access` field of **Thread**(/messaging/customer-chat-api/v3.2/#threads) is now of type `string` and contains reason for access restriction.
```

Comment:
IMO, it should be in the Events category.

Reply:
Copy:

> had the `restricted_access` field type changed to `string`. It contains the reason for access restriction.

Reply:
![image](https://user-images.githubusercontent.com/42515529/89889042-53468600-dbd1-11ea-8196-d2e0af728803.png)